gha/jira/util.py

```diff
@@ -7,6 +7,7 @@
 # endDate=2022-12-02T23:19:00.000+03:00,completeDate=<null>,
 # activatedDate=2022-11-28T23:19:31.860+03:00,sequence=3823,goal=,autoStartStop=false
 AGILE_SPRINT_ID_RE = re.compile(r"\[id=(\d+),")
+PR_NUMBER_RE = re.compile(r"\(#(\d+)\)$")
```

Comment:
После последней скобки может быть ещё текст, например "[skip ci]". Так что ожидать его строго в конце строки некорректно.

```suggestion
PR_NUMBER_RE = re.compile(r"\(#(\d+)\)")
```